### DISCUSSION TITLE
Export beatmap

### DISCUSSION THREAD

**Discussion by Rystars** [ppy/osu#18137] [Ideas]
Add feature to export beatmap like osu skin export feature

**Comment by bdach** [comment]
go to editor -> click file menu -> click "export package"

**Reply by Rystars** [reply]
![Screenshot_2022-07-03-22-05-38-872_sh ppy osulazer](https://user-images.githubusercontent.com/96302690/177045772-0befec8c-ea9f-442c-a315-3f5021067bbd.jpg)

Where? It doesn't exist

**Reply by peppy** [reply]
It won't work on android just yet.
